### ISSUE TITLE
Create a shell script from a Nextflow workflow [question]

### ISSUE THREAD

**Issue by sjackman** [nextflow-io/nextflow#720]
Is it possible to create a shell script from a Nextflow workflow? With a Makefile, I would run…
```sh
make -Bn all >make.sh
```

**Comment by pditommaso** [comment]
No, but the complete sequence of tasks could be recreated via `nextflow log <run name>` plus some bash scripting (mostly for documentation purpose).

**Comment by sjackman** [comment]
Thanks. I'll check that out.
It's quite useful to know what commands are going to be run, before actually running them. Could you please add this feature to your list of feature requests?

**Comment by pditommaso** [comment]
What's exactly your use case? recreate the full execution graph can be quite challenging, not even sure, that's entirely possible.

**Comment by pditommaso** [comment]
> Does Nextflow create this execution plan before it starts running the commands?

No. This is why NF does not have a *dry-run* execution mode. On the other hand this approach allows NF to handle much bigger execution DAG compared to other tools.

**Comment by sjackman** [comment]
Does `triage/needs-information` mean that you'd like more information from me?

**Comment by pditommaso** [comment]
Yes, sorry for the late reply. I mean, the sequence of command can be generated taking the `.command.sh` script in each path returned by the command `nextflow run <name>`. 

Would that work for you and you are expecting a more smooth or different behaviour?

**Comment by pditommaso** [comment]
I'm closing this since there's any more feedback. Feel free to comment below or re-open it if needed.
